pkg/kubelet/cm/cgroup_manager_linux_test.go

```diff
@@ -169,3 +169,55 @@ func TestParseSystemdToCgroupName(t *testing.T) {
 		}
 	}
 }
+
+func TestCpuSharesToCpuWeight(t *testing.T) {
+	testCases := []struct {
+		input    uint64
+		expected uint64
+	}{
+		{
+			input:    0,
```

Comment:
This is now a valid test case. Cpu shares is set to a minimum value of 2 at this time.
https://github.com/kubernetes/kubernetes/pull/102884#discussion_r662552642